common-files/.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @jenkinsci/${artifactId}-plugin-developers
```

Comment:
Is this team created automatically for newly hosted plugins?

Reply:
Yes